src/design/optForceGAMS/optForceWithGAMS.m

```diff
@@ -0,0 +1,840 @@
+    end
+    %print lower and upper bounds, minimum and maximum values for each of
+    %the reactions in wild-type and mutant strain
+    fprintf(freport, '\nLB\tUB\tMin_WT\tMax_WT\tMin_MT\tMax_MT\n');
+    for i = 1:length(model.rxns)
+        fprintf(freport, '%6.4f\t%6.4f\t%6.4f\t%6.4f\t%6.4f\t%6.4f\n', model.lb(i), model.ub(i),...
+            minFluxesW(i), maxFluxesW(i), minFluxesM(i), maxFluxesM(i));
+    end
+    %print target reaction.
+    fprintf(freport, ['\nTarget reaction:\n' targetRxn '\n'] );
+    %print must U set
+    fprintf(freport, '\nMust U Set:\n');
+    for i = 1:length(mustU)
+        fprintf(freport, [mustU{i} '\n']);
+    end
+    %print must L set
+    fprintf(freport, '\nMust L Set:\n');
+    for i = 1:length(mustL)
+        fprintf(freport, [mustL{i} '\n']);
+    end
+    %print constraints
+    fprintf(freport, '\nConstrained reactions:\n');
+    if ~isempty(constrOpt)
+        for i = 1:length(constrOpt.rxnList)
+            fprintf(freport, '%s: fixed in %6.4f\n', constrOpt.rxnList{i}, constrOpt.values(i));
+        end
+    end
+    %print excludad reactions
+    fprintf(freport, '\nExcluded reactions:\n');
+    if ~isempty(excludedRxns)
+        for i = 1:length(excludedRxns.rxnList)
+            fprintf(freport, '%s: Excluded from %s\n', excludedRxns.rxnList{i}, ...
+                regexprep(excludedRxns.typeReg(i), {'U','L','K'}, {'Upregulations','Downregulations','Knockouts'}));
+        end
+    end
+    fprintf(freport, '\nrunID(Main Folder): %s \n\noutputFolder: %s \n\noutputFileName: %s \n',...
+        runID, outputFolder, outputFileName);
+    
+    
+    fprintf(freport, '\nprintExcel: %1.0f \n\nprintText: %1.0f \n\nprintReport: %1.0f \n\nkeepInputs: %1.0f  \n\nkeepGamsOutputs: %1.0f \n\nverbose: %1.0f \n',...
+        printExcel,printText,printReport,keepInputs,keepGamsOutputs,verbose);
+end
+
+%initialize arrays for excluding reactions.
+excludedURxns = {};
+excludedLRxns = {};
+excludedKRxns = {};
+if ~isempty(excludedRxns)
+    for i = 1:length(excludedRxns.rxnList)
+        if strcmp(excludedRxns.typeReg(i), 'U')
+            excludedURxns = union(excludedURxns, excludedRxns.rxnList(i));
+        elseif strcmp(excludedRxns.typeReg(i), 'L')
+            excludedLRxns = union(excludedLRxns, excludedRxns.rxnList(i));
+        elseif strcmp(excludedRxns.typeReg(i), 'K')
+            excludedKRxns = union(excludedKRxns, excludedRxns.rxnList(i));
+        end
+    end
+end
+
+%copy the file for running optForce in GAMS
+copyfile(pathOFG);
+
+%export inputs to GAMS
+inputFolder = 'InputsOptForce';
+exportInputsOptForceToGAMS(model, {targetRxn}, mustU, mustL, minFluxesW, maxFluxesW, minFluxesM, maxFluxesM, k, nSets,...
+    constrOpt, excludedURxns, excludedLRxns, excludedKRxns, inputFolder)
+
+% if the user wants to generate a report, print results.
+if printReport; fprintf(freport, '\n------RESULTS------:\n'); end;
+
+%run optForce in GAMS.
+if verbose
```

Comment:
I am a little bit confusing. In line 522 I see "InputOptForce", not "InputOptForce/". Am I missing something? I tried to make the code independent of the operation system, but I don't know if I succeded.